vipps-login-api.md

```diff
@@ -86,17 +85,13 @@ The user goes to the Vipps app and confirms the login:
 The user is then authenticated in browser and can provide consent if required. Then the user is redirected back to the redirect URI provided by merchant:
 ![Consent in desktop](images/Number_input_flow_desktop2.png	)
 
-#### Phone number based flow - mobile
-This is the fallback flow for mobile devices if it is not possible to use the app-switch based flow. This is the same flow as for desktop above but please note that the user in this flow need to manually navigate back to the browser where they entered the phone number. If the user is remembered in browser the confirmation in app flow will be skipped.
 
-![Phone number based flow - mobile](images/Mobile_flow_with_phone_number.png)
+#### Mobile flow - app switch based flow
+If the user is on a moobile device, the Vipps landing page in the browser will automatically trigger an app switch to the Vipps-app if the user is not rememberd in browsere. The user will not be prompted to enter the phone number. In the Vipps-app the user confirms the login and can choose whether to be rememberd in the browser for later logins. After confirming in app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems. On the Vipps landing page the user will finalise the autentication and provide consents if required. Then the user is redirected back to the redirect URI provided by merchant (can be webpage or app).
```

Comment:
```suggestion
If the user is on a moobile device, the Vipps landing page in the browser will automatically trigger an app switch to the Vipps-app if the user is not rememberd in browsere. The user will not be prompted to enter the phone number. In the Vipps-app the user confirms the login and can choose whether to be rememberd in the browser for later logins. After confirming in the app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems. On the Vipps landing page the user will finalise the autentication and provide consents if required. Then the user is redirected back to the redirect URI provided by merchant (can be webpage or app).
```

Reply:
` After confirming in app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems.`

Synes denne var veldig tung, hva med noe ala
 After confirming in app the user needs to switch back to the Vipps page in the browser/app.

Reply:
Typo på moobile